test/Microsoft.AspNetCore.Authentication.Core.Test/Microsoft.AspNetCore.Authentication.Core.Test.csproj

```diff
@@ -2,7 +2,8 @@
 
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
```

Comment:
Can you check that multi-TFM works in VS? In 15.0-15.2 the project system parses csproj as XML which means it IntelliSense would only recognize one TFM, not two. This is supposedly fixed in 15.3. cref https://github.com/dotnet/project-system/issues/547#issuecomment-277379863.

The workaround is to invert the logic to:
```xml
<TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
<TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
```